web/impact/impact/v1/classes/option_analysis.py

```diff
@@ -96,38 +139,161 @@ def calc_needs(self, option_name):
         }
 
     def calc_needs_distribution(self, option_name):
-        app_ids = self.helper.app_ids_for_feedbacks(self.completed_feedbacks(),
-                                                    option_name=option_name,
-                                                    applications=self.apps)
-        app_counts = Counter(app_ids)
-        counts = Counter(app_counts.values())
-        unread_count = (self.helper.app_count(self.apps, option_name) -
-                        len(app_counts))
+        app_counts = self.application_criteria_read_state(
+            self.completed_feedbacks,
+            option_name=option_name,
+            applications=self.apps)
+        counts = {}
+        for count in app_counts.values():
+            total = count[self.option_spec.criterion.name].get(option_name, 0)
```

Comment:
@VincentHokie I'm getting a fail at this line, `'int' object has no attribute 'get'`

Possibly an unpushed commit?

Reply:
I tested this against a JR without reads as a criteria, it's fixed now